### DISCUSSION TITLE
What is the base amounts of memory usage in the calculation of resources duration

### DISCUSSION THREAD

**Discussion by jayin92** [argoproj/argo-workflows#11427] [Q&A] [answered]
I'm currently working on the resource estimation for Argo Workflow, But I'm encouter some problems. 

In the [document](https://argoproj.github.io/argo-workflows/resource-duration/), the base amounts of memory is `1Gi`. However, in the web UI, it shows that the unit of memory is `100Mi`. Also, the calculation in the document is also really weird. The example used 100Mi memory for 1 min, but the final resources duration is 18sec * (100Mi). Should this be 18sec * (1Gi)?

In conclusion, I have two questions:

1. What is the actual base amounts for memory when calculating resources duration
2. Is the calculation in document wrong

<img width="429" alt="image" src="https://github.com/argoproj/argo-workflows/assets/11909159/5d5c2a3c-1906-4adf-83ae-9217134b1eaf">

**Comment by agilgur5** [comment, marked as answer]
Addressed this [over Slack](https://cloud-native.slack.com/archives/C01QW9QSSSK/p1690186886151779?thread_ts=1690167353.854669&cid=C01QW9QSSSK).

I searched through the source code and found that the base amount / denominator for memory is [`100Mi`](https://github.com/argoproj/argo-workflows/blob/7f22085d16316bdd77bdb1060455140d9d6e1664/pkg/apis/workflow/v1alpha1/workflow_types.go#L2012). This means that this documentation page was indeed incorrect.

The docs have now been fixed in #11432 🙂
